### ISSUE TITLE
LinkifyFilter does not use default callbacks

### ISSUE THREAD

**Issue by rixx** [mozilla/bleach#538]
The documentation [says](https://bleach.readthedocs.io/en/latest/linkify.html#bleach.linkifier.LinkifyFilter) that `LinkifyFilter` works just like `linkify`, including adding the `rel=nofollow` attribute by default.

This does not appear to work for me. If I use `bleach.clean` with plain `LinkifyFilter` as described in the documentation, the default filters are not applied. I have to add them explicitly instead.

Minimal example:

```python
from functools import partial
import bleach

bleach.Cleaner(
    filters=[
        partial(
            bleach.linkifier.LinkifyFilter,
            skip_tags=["pre"],
        )
    ]
).clean("example.com")

'<a href="http://example.com">example.com</a>'


bleach.Cleaner(
    filters=[
        partial(
            bleach.linkifier.LinkifyFilter,
            skip_tags=["pre"],
            callbacks=bleach.linkifier.DEFAULT_CALLBACKS,  # add default callbacks
        )
    ]
).clean("example.com")

'<a href="http://example.com" rel="nofollow">example.com</a>'
```

**Comment by codebykat** [comment]
Just noting I had this same problem in Bleach 2.0.0, and it persists after an upgrade to 3.1.5. The default callbacks are applied if the linker is called directly rather than as a filter. I'm using the same workaround as the above.

**Comment by g-k** [comment]
Fixed in v3.2.1 (PR https://github.com/mozilla/bleach/pull/555).
